enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -867,18 +864,27 @@ func (c *FirecrackerContainer) LoadSnapshot(ctx context.Context) error {
 		return err
 	}
 
+	ctx, cancel := c.monitorVMContext(ctx)
```

Comment:
Could you add a comment that the context used to start the VM _should_ be different than the cancellable one used to monitor the context?